README.md

```diff
@@ -8,35 +8,54 @@ Repository to accompany our publications
 ## Requires 
 - Flexfringe (https://github.com/tudelft-cda-lab/FlexFringe)
 - Python packages
-  - Graphviz
-  - Seaborn
-  - Requests
+  - `graphviz`
+  - `requests`
+  - `numpy`
+  - `matplotlib`
   - [Optional] Fastai (1.0.58)
   - [Optional] Spacy 2.3.5
 
 **Or you can switch to the `docker` branch for a docker container.**
 
 
 ## Usage
-`python sage.py {path/to/json/files} {experiment-name} {alert-filtering-window} {alert-aggr-window} {(start_hour,end_hour)}`
-
-- `{path/to/json/files}`: folder containing intrusion alerts in json format. `sample-input.json` provides an example of accepted file format. 
-> Ideal setting: One json file for each attacker/team. Filename considered as attacker/team label. 
-- `{experiment_name}`: custom name for all artefacts
-> Figures, trace files, model files, attack graphs are saved with this prefix for easy identification. 
-- `{alert_filtering_window}`: time window in which duplicate alerts are discarded (default: 1.0 sec)
-- `{alert_aggr_window}`: aggregate alerts occuring in this window as one episode (default: 150 sec)
-- [Optional] `{(start_hour,end_hour)}`: Time range (in hours). A floating-point tuple limiting the alerts that are parsed and involved in the final attack graphs. 
+`python sage.py path_to_json_files experiment_name [-h] [-t T] [-w W] [--timerange TIMERANGE TIMERANGE] [--dataset {cptc,other}] [--keep-files]`
+
+Required positional arguments:
+
+* `path_to_json_files`: Directory containing intrusion alerts in json format. `sample-input.json` provides an example of the accepted file format.
+> Ideal setting: One json file for each attacker/team. Filename considered as attacker/team label.
+* `experiment_name`: Custom name for all artefacts.
+> Figures, trace files, model files, attack graphs are saved with this prefix for easy identification.
+
+Options:
+
+* `-h`, `--help`: Show the help message and exit.
+* `-t`: Time window in which duplicate alerts are discarded (default: 1.0 sec).
+* `-w`: Aggregate alerts occuring in this window as one episode (default: 150 sec).
+* `--timerange`: A floating-point tuple limiting the alerts that are parsed and involved in the final attack graphs (default: (0, 100)).
 > If not provided, the default values of (0,100) are used, meaning alerts from 0-th to 100-th hour (relative to the start of the alert capture) are parsed.
+* `--dataset`: The name of the dataset with the alerts (default: other, available options: cptc, other).
```

Comment:
Even though we have discussed that `other` is the default value, are you sure you want to keep it this way? During RP, we were only using both CPTC datasets, because of which it might be inconvenient to always type this option. On the other hand, SAGE has not been tested yet on other datasets apart from CCDC, so for now "other" is essentially only CCDC.

Reply:
Yep, it's fine like this